### ISSUE TITLE
Annoying window appears on Windows 10 node 14

### ISSUE THREAD

**Issue by deliverymanager** [cristiammercado/node-disk-info#16]
Please place the windowsHide: true attribute

```
        return execSync(command, {
            windowsHide: true
        }).toString();
```

**Comment by deliverymanager** [comment]
I have tested it in my own fork and it works

**Comment by Ekristoffe** [comment]
fix added in my last PR (#15). hope @cristiammercado will merge it soon ...

**Comment by Ekristoffe** [comment]
Don't worry.
With what happening in the world I understand that people have other priorities (families and trying to survive).

**Comment by cristiammercado** [comment]
Hi,

This issue has been fixed in [v1.3.0](https://github.com/cristiammercado/node-disk-info/releases/tag/v1.3.0) already released in [npm](https://www.npmjs.com/package/node-disk-info). I proceed to close it.
